ow_faults/src/FaultInjector.cpp

```diff
@@ -26,6 +29,20 @@ void FaultInjector::faultsConfigCb(ow_faults::FaultsConfig& faults, uint32_t lev
   m_faults = faults;
 }
 
+void FaultInjector::setSytemFaultsMessage(ow_faults::SystemFaults& msg, int value) {
+  // for now only arm execution errors
+  msg.header.stamp = ros::Time::now();
+  msg.header.frame_id = "/world";
+  msg.value = value; //should be 4 for now
+}
+
+void FaultInjector::setArmFaultMessage(ow_faults::ArmFaults& msg, int value) {
+  // for now only arm execution errors
+  msg.header.stamp = ros::Time::now();
+  msg.header.frame_id = "/world";
+  msg.value = value; //should be 1 for now
+}
+
```

Comment:
Even in the comments, use the new enum names instead of 1 and 4.

Reply:
done!